forester/src/photon_indexer.rs

```diff
@@ -216,4 +223,32 @@ impl<R: RpcConnection> Indexer<R> for PhotonIndexer<R> {
 
         Ok(proofs)
     }
+
+    async fn get_multiple_new_address_proofs_full(
```

Comment:
the name seems to be a bit outdated since we return trees of height 40 now instead of full proofs of height 26.
Could rename to for example:
```suggestion
    async fn get_multiple_new_address_proofs_h40(
```